client/lib/abtest/active-tests.js

```diff
@@ -146,4 +146,13 @@ export default {
 		},
 		defaultVariation: 'originalFlavor',
 	},
+	onlyJetpackMonthly: {
+		datestamp: '20190212',
+		variations: {
+			original: 50,
+			monthlyOnly: 50,
+		},
+		defaultVariation: 'original',
+		countryCodeTargets: [ 'ES', 'IT', 'PT', 'FR', 'NL', 'DE', 'BE', 'PL', 'SE' ],
```

Comment:
I think it would be useful to get some background on the selection of countries for this test.

Reply:
I'll provide that in a post about it :+1: